src/test/unit/helperFunctions/test_rest.py

```diff
@@ -64,40 +67,46 @@ def test_convert_rest_request_succeeds(data):
 
 
 def test_get_recursive():
-    assert not get_recursive(None)
+    assert not get_recursive_flag(None)
 
     with pytest.raises(ValueError):
-        get_recursive(dict(recursive='bad_string'))
+        get_recursive_flag(dict(recursive='bad_string'))
 
     with pytest.raises(ValueError):
-        get_recursive(dict(recursive='2'))
+        get_recursive_flag(dict(recursive='2'))
 
-    no_flag = get_recursive(dict())
+    no_flag = get_recursive_flag(dict())
     assert not no_flag
 
-    false_result = get_recursive(dict(recursive='false'))
+    false_result = get_recursive_flag(dict(recursive='false'))
     assert not false_result
 
-    good_result = get_recursive(dict(recursive='true'))
+    good_result = get_recursive_flag(dict(recursive='true'))
     assert good_result
 
 
-def test_get_summary_flag():
-    assert not get_summary_flag(None)
+@pytest.mark.parametrize('get_request_parameter_function, name', [
+    (get_recursive_flag, 'recursive'),
+    (get_inverse_flag, 'inverse'),
+    (get_summary_flag, 'summary'),
+    (get_tar_flag, 'tar'),
+])
+def test_get_boolean_from_request(get_request_parameter_function, name):
+    assert not get_request_parameter_function(None)
 
     with pytest.raises(ValueError):
-        get_summary_flag(dict(summary='bad_string'))
+        get_request_parameter_function({name: 'bad_string'})
 
     with pytest.raises(ValueError):
-        get_summary_flag(dict(summary='2'))
+        get_request_parameter_function({name: '2'})
```

Comment:
Redundant? (I know it's not coming from you)

Reply:
I thougt so too but kept it anyway -> removed